pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/JstlEvaluator.java

```diff
@@ -81,6 +81,12 @@ private void registerFunctions() {
             "timestampAdd",
             JstlFunctions.class.getMethod(
                 "timestampAdd", Object.class, Object.class, Object.class));
+    this.expressionContext
+        .getFunctionMapper()
+        .mapFunction(
+            "fn",
+            "decimal",
+            JstlFunctions.class.getMethod("toBigDecimal", byte[].class, int.class));
```

Comment:
We should implement coercion to BigInteger so you can take Object as first arg.
Also we should use Object for the second arg and coerce to Integer.
And to be complete it would be nice to have toBigDecimal(Object) that coerces the first arg to Double.